tools/monitors/cli-monitor/test/cli.test.ts

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { expect } from 'chai';
+import { exec } from 'node:child_process';
+import { promisify } from 'node:util';
+
+describe('CLI', () => {
+  it('prints config', async () => {
```

Comment:
I assume the intention is to eventually test other parts of the CLI, right? Even so, I'm not sure if I understand the rationale for making this a separate package, shouldn't this be a test of the CLI package?

Reply:
> I assume the intention is to eventually test other parts of the CLI, right?

Correct

> Even so, I'm not sure if I understand the rationale for making this a separate package, shouldn't this be a test of the CLI package?

The reason it's a separate package is so that it can live outside of the pnpm workspace and install our `@dxos` dependencies directly from npm. The goal of these monitor tests now, as I see it, is to install deps from npm after a release goals and make sure they function as we intend.

Reply:
Makes perfect sense. Perhaps it'd be good to capture the intent and how you expect it be invoked in the README?